app/src/main/java/com/sunasterisk/travelapp/data/models/Location.kt

```diff
@@ -106,7 +109,7 @@ data class Location(
                     val locationJSONObject = jsonObject.getJSONObject(LOCATION_OBJECT)
                     var description: String
                     var propertiesNumber = 0
-                    if (TYPE_GEO == type) {
+                    if (TYPE_GEO.equals(type)) {
```

Comment:
== vs equals có gì khác nhau nhỉ?

Reply:
== so sánh địa chỉ của biến còn equals so sánh value

Reply:
Trong java thì đúng rồi nhá =)) https://stackoverflow.com/questions/7520432/what-is-the-difference-between-and-equals-in-java
Xem lại trong Kotlin đi nha
Code kiểu kia là các thầy trên trường ưng rồi nhưng mà đúng với java thôi nhá

Reply:
vâng ạ do em chưa tìm hiểu kỹ == trong kotlin ạ